projects/ion/src/lib/avatar/defaultImage.directive.ts

```diff
@@ -0,0 +1,23 @@
+import { Directive, HostBinding, HostListener, Input } from '@angular/core';
+
+@Directive({
+  selector: 'img[default]', // eslint-disable-line
```

Comment:
Don't you think would be better to follow the pattern and use the ion prefix?
something like:
```suggestion
  selector: 'ion-img[default]', 
```

Reply:
With the ion prefix it was not possible to use the directive, so the line is being ignored